bcoin-org/bcoin
# Node RPC Tests

RPC tests:
- Fix `help help`
- `help` tests
-  remove `getinfo`.
- test for `getblockchaininfo`.
- test for `getnetworkinfo`.
- Change `verifytxoutproof` to check totalTX count if possible. (performance degrades)
  - If performance is an issue we could index this information separately or with headers(entry).
- Add tests for `gettxoutproof` and `verifytxoutproof`.
